### PR TITLE
Remove piercing from Felinid melee

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR 
Removes the Piercing damage from the Felinid claw melee as currently most of our melee weapons were modified to no longer have that damage type. Something about it being the gun damage type.

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Felinids no longer do piercing damage with their claws.

### CODE REVIEW

File: Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml [hunk 1]
```diff
@@ -123,7 +123,6 @@
       types:
         Blunt: 1
         Slash: 4
-        Piercing: 1
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml]: This has the consequence of reducing total damage.

Comment (FluffiestFloof) [reply Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml]: Which was already 1 more damage than any other morphotype (except oni)
I can add it to blunt if it's really an issue that Felinids don't deal 6 damage anymore as the default damage is 5 blunt

Comment (Torun-S) [reply Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml]: Yeah, add it to Blunt, I'd say.

Comment (Vordenburg) [reply Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml]: Given how flat damage reduction works, I'd suggest putting it into slash instead.

**Pull request merged**